### ISSUE TITLE
[Out of context request] Please hear me out - Would it be possible for an offline "Potplayer" version of this helpful tool?

### ISSUE THREAD

**Issue by 4Brad-23** [meetDeveloper/Dictionary-Anywhere#61]
"Potplayer" (Media player) allows subtitle word selection and with a tool like yours, it'll be very educational and close to a miracle. Thank you.

**Comment by hashim-cpro** [comment]
Agreed, it could use a local dictionary and browser based audio features. It would become fast as well this way.
